src/components/raycaster.js

```diff
@@ -1,11 +1,19 @@
+/* global MutationObserver */
+
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
 
 var bind = utils.bind;
+var warn = utils.debug('components:raycaster:warn');
 
 var dummyVec = new THREE.Vector3();
 
+// Defines selectors that should be 'safe' for the MutationObserver used to
+// refresh the whitelist. Less common selectors, like [position=0 2 0], cannot
+// be detected here.
+var SAFE_SELECTOR_RE = /^[\w\s-.[\]#]+$/;
```

Comment:
Can explain more what this selector matches, and name maybe like `OBSERVER_SELECTOR_RE`